app/AppKernel.php

```diff
@@ -35,11 +35,19 @@ public function getRootDir()
 
     public function getCacheDir()
     {
+        if (in_array($this->environment, ['dev', 'test'])) {
```

Comment:
Thank you @shouze for your nice tips!
I am still pretty new to Docker (I have more advanced uses of it in real projects though), so if you have any link to an example showing an implementation of log redirection for a Symfony app in a Docker container that would be great! I will otherwise dig into it.